packages/renderer/src/lib/preferences/PreferencesExtensionList.svelte

```diff
@@ -116,11 +119,11 @@ async function updateExtension(extension: ExtensionInfo, ociUri: string) {
         </div>
       </div>
     </div>
-    <div class="bg-charcoal-600 mt-5 rounded-md p-3">
+    <div class="bg-charcoal-600 mt-5 rounded-md p-3" role="region" aria-label="InstalledExtensions">
       <table class="min-w-full">
         <tbody>
           {#each $extensionInfos as extension}
-            <tr class="border-y border-gray-900">
+            <tr class="border-y border-gray-900" aria-label="{extension.name}">
```

Comment:
this is nice

Reply:
But I am missing a descriptions/labels/roles in each cell that the row contains. if we would have a way to get the information from each cell, that would be nice.